dnf-repoconfig-daemon/example_client/client.cpp

```diff
@@ -0,0 +1,30 @@
+#include <sdbus-c++/sdbus-c++.h>
+#include <vector>
+#include <string>
+#include <iostream>
+#include <unistd.h>
+
+int main(int argc, char *argv[])
+{
+    auto connection = sdbus::createSystemBusConnection();
+    std::cout << "Client name: " << connection->getUniqueName() << std::endl;
+    const char* destinationName = "org.rpm.dnf.v1.rpm.RepoConf";
+    const char* objectPath = "/org/rpm/dnf/v1/rpm/RepoConf";
+    auto repoconf_proxy = sdbus::createProxy(*connection, destinationName, objectPath);
+    const char* interfaceName = "org.rpm.dnf.v1.rpm.RepoConf";
+    repoconf_proxy->finishRegistration();
+
+    {
+        std::vector<std::string> ids = {"fedora", "updates"};
+        std::vector<std::map<std::string, sdbus::Variant>> repolist;
```

Comment:
These types are used everywhere.
Couldn't we name them to make the definitions shorter?

std::map<std::string, sdbus::Variant> -> KeyValueMap
std::vector<std::map<std::string, sdbus::Variant>> -> KeyValueMapList
Or something similar...